### DISCUSSION TITLE
Using Liqo without network features

### DISCUSSION THREAD

**Discussion by EladDolev** [liqotech/liqo#2300] [Q&A] [answered]
Hello there

Super excited about your product !

I was wondering whether it can be used without any network features e.g. service reflection etc.
We have a very simple use case where we have access to kube-apiserver between clusters, but we don't want to connect the networks in any way or expose a load balancer.
We only want to be able to offload workloads, optionally together with other basic resources such as CM and we have no need for any kind of communication between pods.

I was trying all kinds of setups but I'm unable to use `liqoctl` to peer the clusters

Is there any way to achieve this, even in a hacky way ?

Thanks !

**Comment by fra98** [comment, marked as answer]
Hi @EladDolev,

You can completely disable the network fabric, as described [here](https://docs.liqo.io/en/v0.10.1/usage/external-network.html#enable-disable-the-external-network). Although you still need to authenticate the two clusters, which requires exposing the `liqo-auth` service (with NodePort, LoadBalancer or Ingress).

Regarding the need to reflect only some types of resources, you can configure your custom reflection policies. Basically, for each resource reflector, you can either disable it (scaling the workers to 0) or choose between a white-list or black-list reflection policy. You can find more info [here](https://docs.liqo.io/en/v0.10.1/usage/reflection.html#reflection-policies).
